src/App.js

```diff
@@ -23,23 +29,26 @@ export default function App() {
   return (
     <Router>
       <Routes>
-        <Route
-          path="/"
-          element={
-            <Onboarding1 />
-            // <h2>Just localhost</h2>
-          }
-        />
-        <Route path="/registry" element={<Registry />} />
-        <Route path="/registrystart" element={<RegistryStart />} />
-        <Route path="/registryadd" element={<RegistryAdd />} />
-        <Route path="/guest-invite" element={<Invite />} />
-        <Route path="/rsvp-form" element={<RsvpForm />} />
-        <Route path="/browse-theme" element={<BrowseTheme />} />
-        <Route path="/choose-content" element={<ChooseContent />} />
-        <Route path="/invitation-form" element={<InvitationForm />} />
-        <Route path="/rsvp-preview" element={<RSVP />} />
-        <Route path="/guestlist" element={<GuestListMainPage />} />
+        <Route exact path="/onboarding-1" element={<Onboarding1 />} />
+        <Route exact path="/onboarding-2" element={<Onboarding2 />} />
+        <Route exact path="/onboarding-3" element={<Onboarding3 />} />
+        <Route exact path="/onboarding-4" element={<Onboarding4 />} />
+        <Route exact path="/onboarding-5" element={<Onboarding5 />} />
+        <Route path="/" element={<Frame />}>
+          <Route path="/dashboard" element={<Dashboard />} />
```

Comment:
might need to move guest-invite and rsvp-form paths one level up so they dont have the header/footer. not an immediate change needed